services/s3/src/main/java/software/amazon/awssdk/services/s3/checksums/S3ChecksumValidatingInputStream.java

```diff
@@ -25,7 +25,7 @@
 import software.amazon.awssdk.utils.BinaryUtils;
 
 @SdkInternalApi
-public class ChecksumValidatingInputStream extends InputStream implements Abortable {
+public class S3ChecksumValidatingInputStream extends InputStream implements Abortable {
```

Comment:
How come this is not under internal subpackage... Can we do a quick check to see if anyone is using this class internally?